packages/web-components/src/components/combo-button/combo-button.ts

```diff
@@ -0,0 +1,154 @@
+   * Specify whether the ComboButton should be disabled, or not.
+   */
+  @property({ type: Boolean, reflect: true })
+  disabled?;
+
+  /**
+   * Provide the label to be rendered on the primary action button.
+   */
+  @property()
+  label!: string;
+
+  /**
+   * Experimental property. Specify how the menu should align with the button element
+   */
+  @property({ reflect: true, type: String, attribute: 'menu-alignment' })
+  menuAlignment = POPOVER_ALIGNMENT.TOP;
+
+  /**
+   * Provide an optional function to be called when the primary action element is clicked.
+   */
+  @property({ type: Function })
+  onClick?: (event: Event) => void;
+
+  /**
+   * Specify the size of the button and menu.
+   */
+  @property({ type: COMBO_BUTTON_SIZE, reflect: true })
+  size = COMBO_BUTTON_SIZE.LARGE;
+
+  /**
+   * Specify how the trigger tooltip should be aligned.
+   */
+  @property({ reflect: true, attribute: 'tooltip-alignment' })
+  tooltipAlignment = ICON_BUTTON_TOOLTIP_ALIGNMENT.TOP;
+
+  @HostListener('click')
+  // @ts-ignore: The decorator refers to this method but TS thinks this method is not referred to
+  private _handleClick = (event: Event) => {
+    const path = event.composedPath();
+    if (path.includes(this._triggerNode)) {
+      this._open = !this._open;
+    } else if (this._open) {
+      this._open = false;
+    }
+  };
+
+  @HostListener('focusout')
+  // @ts-ignore: The decorator refers to this method but TS thinks this method is not referred to
+  private _handleBlur = ({ relatedTarget }: FocusEvent) => {
+    // Close the menu if the focus moves outside the menu button or menu
+    if (!this.contains(relatedTarget as Node)) {
+      this._open = false;
+    }
+  };
+
+  updated(changedProperties) {
+    const menu = this.querySelector(`${prefix}-menu`) as CDSMenu;
+
+    if (
+      changedProperties.has('_open') ||
+      changedProperties.has('menuAlignment')
+    ) {
+      this.updateComplete.then(() => {
+        const styleElement = menu.shadowRoot?.querySelector(
+          `.${prefix}--menu`
+        ) as HTMLElement;
+
+        menu.open = this._open;
+
+        this._menuController.setPlacement({
+          trigger: this,
+          target: menu,
+          alignment: this.menuAlignment,
+          styleElement,
+          matchWidth: true,
+          open: this._open,
+        });
+      });
+    }
+
+    if (changedProperties.has('size')) {
+      menu.setAttribute('size', this.size);
+    }
+  }
+
+  render() {
+    const { size, disabled, label, tooltipAlignment, menuAlignment, onClick } =
+      this;
+    return html`
+      <cds-button size=${size} ?disabled=${disabled} @click=${onClick}>
+        ${label}
+      </cds-button>
+      <cds-icon-button
+        size=${size}
+        ?disabled=${disabled}
+        align=${tooltipAlignment}
+        menu-alignment=${menuAlignment}
+        part="trigger">
+        ${ChevronDown16({ slot: 'icon' })}
+        <span slot="tooltip-content">Additional actions</span>
```

Comment:
Since we don't have a translation like react: https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/ComboButton/index.tsx#L32-L34

Maybe we can just have a prop instead called maybe `tooltip-content` which accepts a string, but the default can be `Additional actions`.